pico-w-blink-sdk/BridgingHeader.h

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors.
```

Comment:
Could you update the year to 2024 for the files added?

Reply:
@rauhul I fixed it. https://github.com/apple/swift-embedded-examples/pull/11/commits/ba8f899073e86b0e49c145c113d0d9c991df4122